packages/monolith/components/Icons/EthereumIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import { createIcon } from "@chakra-ui/react";
+
+export default createIcon({
```

Comment:
Instead of doing this use [this library](https://www.npmjs.com/package/react-crypto-icons)